build_carddb_lorcana.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""
+This script downloads the latest Disney Lorcana card data from https://lorcanajson.org/ and processes
+it to turn the highly-structured data there into a flat list of card names to descriptions
+formatted to send down the chat.
+"""
+
+import common
+common.FRAMEWORK_ONLY = True
+
+import json
+import sys
+import zipfile
+
+import psycopg2
+
+from common.card import CARD_GAME_LORCANA
```

Comment:
For the `filteredname` change below:
```suggestion
from common.card import clean_text, CARD_GAME_LORCANA
```